airflow/migrations/versions/004c1210f153_increase_queue_name_size_limit.py

```diff
@@ -40,7 +40,7 @@ def upgrade():
     by broker backends that might use unusually large queue names.
     """
     # use batch_alter_table to support SQLite workaround
-    with op.batch_alter_table('task_instance') as batch_op:
+    with op.batch_alter_table('task_instance') as batch_op:  # pylint:disable=no-member
```

Comment:
This change seems unrelated?

Reply:
When I switched pylint to use docker image - that problem started to show up in Travis. Seems to come up with the way pylint was run in the non-contenerised version - skipping migrations. I think it's good to keep this fix so that we have green CI. One of my unrelated changes that I am going to submit soon is to disable some pylint checks on migrations. But it would be great to merge this one so that we have consistent environment for pylint checks. It was one of the problems so far that different environments produced different set of pylint errors. By implementing this change we will have one environment for both - local and CI.

Reply:
This file should have been skipped from pylint though:

https://github.com/apache/airflow/blob/722379adc7a51bfa19d08b74ce8f048c85aa6738/scripts/ci/pylint_todo.txt#L238

Reply:
Ah. I know then. I tested it with pre-commit hook which does not take into account the exclusions (it works on all locally modified files). I think I touched the file locally before. I will remove it and try to see if it is still OK.

Reply:
Fixed all that pylint fixes that were not needed.